web/README.md

```diff
@@ -0,0 +1,34 @@
+This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+
+## Getting Started
+
+First, run the development server:
+
+```bash
+npm run dev
+# or
+yarn dev
+# or
+pnpm dev
```

Comment:
which one do you use?

I recommend staying on top of this doc and documenting *everything*.  Every command you ran to install or run the server, to update, etc.  Basically I try to think of this doc as "what if I had to set up all over again on a new computer" and allow your future self to do 0 thinking.

Reply:
Here's what I have going in the backend/ dir https://github.com/sllewely/recommendations/tree/main/backend